backend/module/eCampCore/src/Service/InvitationService.php

```diff
@@ -116,4 +105,32 @@ public function rejectInvitation(string $inviteKey): Invitation {
 
         return new Invitation($camp->getId(), $camp->getTitle(), null, null);
     }
+
+    /**
+     * @throws EntityNotFoundException
+     * @throws NoAccessException
+     * @throws NotAuthenticatedException
+     * @throws EntityValidationException
+     */
+    public function resendInvitation(string $campCollaborationId): Invitation {
+        /** @var CampCollaboration $campCollaboration */
+        $campCollaboration = $this->campCollaborationRepository->findByCampCollaborationId($campCollaborationId);
```

Comment:
Geht das auch? 
`$campCollaboration = $this->campCollaborationRepository->find($campCollaborationId)`

Reply:
Ja, geht, habe ich geändert